src/openforms/registrations/contrib/zgw_apis/tests/test_utils.py

```diff
@@ -0,0 +1,75 @@
+from django.test import TestCase
```

Comment:
this doesn't seem to do DB queries, so you can use `SimpleTestCase` instead which is faster :)